niworkflows/interfaces/mni.py

```diff
@@ -482,7 +482,7 @@ def create_cfm(in_file, lesion_mask=None, global_mask=True, out_path=None):
         lm_img = nb.as_closest_canonical(nb.load(lesion_mask))
 
         # Subtract lesion mask from secondary mask, set negatives to 0
-        data = np.fmax(data - lm_img.get_data(), 0)
+        data = np.fmax(data - lm_img.get_fdata(), 0)
```

Comment:
Numpy will coerce to `np.array`.

```suggestion
        data = np.fmax(data - lm_img.dataobj, 0)
```